deployment/package-registry.yml

```diff
@@ -1,5 +1,7 @@
 package_paths:
-  - /packages
+  - /packages/snapshot
```

Comment:
In the PR https://github.com/elastic/package-registry/pull/546  there was a rule introduced: package is served from the first found path. Not sure why are you proposing the reversed order.

Reply:
Also if you modify the location of default packages in the package-registry, please review "testing/environment" if it still works with the update.

Reply:
Not proposing a new order, my testing just showed it the other way around. But let me test again, will report back.

As soon as I decomission master, will update also the package-registry.

Reply:
Let me know what is the order. I assume that if it's different from what was intended, it's a bug to be fixed.

Reply:
I did a short research and I think there is a bug in `/search`. It overwrites existing packages once it leaves the double for:

```
			if !all {
				log.Println("packageList:", len(packagesList))
				// Check if the version exists and if it should be added or not.
				for _, versions := range packagesList {
					for _, pp := range versions {
						if pp.Name == p.Name {

							// If the package in the list is newer or equal, do nothing.
							if pp.IsNewerOrEqual(p) {
								continue
							}

							log.Println(pp.Name, pp.Path)

							// Otherwise delete and later add the new one.
							delete(packagesList[pp.Name], pp.Version)
						}
					}
				}
			}

			if _, ok := packagesList[p.Name]; !ok {
				packagesList[p.Name] = map[string]util.Package{}
			}
			packagesList[p.Name][p.Version] = p
```

Reply:
I see. I think what we have here is ok but we should adjust getPackages so it takes care of the merging. This means what `getPackages` returns already only contains the packages to be served.

Reply:
I opened a quick fix here: https://github.com/elastic/package-registry/pull/575

Reply:
Updated dependency to get most recent registry here and adjusted the path order.